### ISSUE TITLE
Make imagemagick dependency only for production build

### ISSUE THREAD

**Issue by adborden** [caciviclab/odca-jekyll#144]
We recently added `mini_magic` to handle automated re-sizing for candidate photos (and other assets). I was thinking the resized images would only show up in the production build,  but everyone is running into this error `Liquid Exception: ImageMagick/GraphicsMagick is not installed in ...`.

**Comment by ckingbailey** [comment]
Is this resolved by #201?

**Comment by adborden** [comment]
No, this is still valid. We want to remove the ImageMagick dependency from development.
